modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -23,11 +23,12 @@ config TFM_BOARD
 	  TFM has been ported to the given board/SoC.
 
 menuconfig BUILD_WITH_TFM
-	bool "Build with TF-M as the Secure Execution Environment"
+	bool "Build with TF-M as the Secure Execution Environment [EXPERIMENTAL]"
```

Comment:
It doesn't make sense to me to mark TF-M integration as experimental at this point.